src/sample-data.js

```diff
@@ -0,0 +1,23 @@
+const answers = {
```

Comment:
This file is also unused as far as I can tell, so please remove if no longer needed to keep your repo clean.